update

```diff
@@ -219,6 +219,8 @@ def main(create_org_dir, delete_orphaned):
                 raise
 
     q = queue.Queue()
+    if concurrency:
+        num_worker_threads = concurrency
     num_worker_threads = multiprocessing.cpu_count()
```

Comment:
Oh, indeed, thanks. I was wrong. 
I was doing some other things for the script. So, I needed to split the patch. But I mistook it... :-p